pkgs/development/compilers/llvm/7/libcxxabi/default.nix

```diff
@@ -21,6 +22,13 @@ stdenv.mkDerivation {
     export TRIPLE=x86_64-apple-darwin
   '' + lib.optionalString stdenv.hostPlatform.isMusl ''
     patch -p1 -d $(ls -d libcxx-*) -i ${../../libcxx-0001-musl-hacks.patch}
+  '' + lib.optionalString (!stdenv.cc.isClang) ''
+    pushd libcxx-*
```

Comment:
is it possible to have more than one `libcxx-*` library?

Reply:
No, it's created by the unpackFile above